### DISCUSSION TITLE
Remaining issues for Ranked Score recalculation

### DISCUSSION THREAD

**Discussion by WitherFlower** [ppy/osu#27007] [Q&A]
Follow up to https://github.com/ppy/osu/discussions/26960#discussioncomment-8356963

Here is a list of all the things that need to be taken into consideration before recalculating ranked score : 
- Any changes to mod multipliers will make the ranked score totals obsolete and need a recalculation unless the totals can be made "dynamic", but I doubt that is feasible 
- It is possible to abuse Strict Tracking to gain more score than expected, see https://github.com/ppy/osu/issues/19232
- It is possible to play a map on lazer then on stable to gain ranked score twice, see https://github.com/ppy/osu/discussions/26890. Regarding this particular issue, I'm not sure if stable and lazer should have different ranked score totals or stable's "score uploader" should be updated to use lazer classic scoring. If stable does get updated, meaning the number you see on the result screen is different from the actual Ranked Score increase, that is not a problem with the scorefarming community.

**Comment by respektive** [comment]
One more thing I noticed is, that when you're overwriting stable scores on lazer it is possible to miss out on Ranked Score. This is, because the Classic Score can be higher than the SV1 values (It's higher most of the time for chokes).

Let's say you have a score set on stable that is 50 million points in ScoreV1, but converted to Classic Score it's 100 million. Now on lazer you overwrite that score with a new one being worth 110 million. What happens is you will only get a net gain of 10 million points, while you should have gained 60 million points.

I think updating the stable "score uploader" and recalculating all scores to use the Classic Scoring, like suggested above, seems like a good solution.
